lib/s3_meta_sync/syncer.rb

```diff
@@ -210,12 +222,12 @@ def download_meta(destination)
       parse_yaml_content(content)
     rescue OpenURI::HTTPError
       retries ||= 1
-      if retries == 2
-        raise RemoteWithoutMeta
-      else
+      if retries <= 1
```

Comment:
`retries` starts at 1, gets incremented in the next line, and if there is another exception, it'll hit the else condition